packages/config/config/devices/0x0090/templates/kwikset_template.json

```diff
@@ -276,5 +276,143 @@
 				"value": 1
 			}
 		]
+	},
+	"alarm_map_idle": {
+		"from": {
+			"alarmType": 0 // No status reported
+		},
+		"to": {
+			"notificationType": 6, // Access Control
+			"notificationEvent": 0 // idle
+		}
+	},
+	"alarm_map_lock_jammed": {
+		"from": {
+			"alarmType": 9 // Lock Jammed
+		},
+		"to": {
+			"notificationType": 6, // Access Control
+			"notificationEvent": 11 // Lock state: Lock Jammed
+		}
+	},
+	"alarm_map_keypad_unlocked": {
+		"from": {
+			"alarmType": 16 // Keypad Unlock
+		},
+		"to": {
+			"notificationType": 6, // Access Control
+			"notificationEvent": 6, // Keypad unlock operation
+			"eventParameters": {
+				"userId": "alarmLevel"
+			}
+		}
+	},
+	"alarm_map_keypad_lock": {
+		"from": {
+			"alarmType": 18 // Keypad Lock
+		},
+		"to": {
+			"notificationType": 6, // Access Control
+			"notificationEvent": 5, // Keypad lock operation
+			"eventParameters": {
+				"userId": "alarmLevel"
+			}
+		}
+	},
+	"alarm_map_keypad_unlocked": {
+		"from": {
+			"alarmType": 19 // Keypad Unlock
+		},
```

Comment:
Duplicate key (alarmType 16 is called the same)

Reply:
I didn't notice this because it is in your table that way. Where did you get that from?

![Screen Shot 2021-03-13 at 12 06 29 AM](https://user-images.githubusercontent.com/33612110/111019883-05287980-8390-11eb-9576-ea6d0486b3c8.png)

Reply:
In the Issue 16 is a Yale value, should that be dropped from the kwikset template altogether?